plugins/clipboard-manager/ios/Sources/ClipboardPlugin.swift

```diff
@@ -38,6 +38,16 @@ class ClipboardPlugin: Plugin {
       invoke.reject("Clipboard is empty")
     }
   }
+    
+  @objc public func clear(_ invoke: Invoke) throws {
+      let clipboard = UIPasteboard.general
+      if clipboard.items.isEmpty {
+          invoke.reject("Clipboard is empty")
```

Comment:
I don't think we really need to return with an Error here, we should resolve with success if it is already cleared